module/Education/view/education/education/course.phtml

```diff
@@ -6,86 +6,96 @@ $this->headTitle($this->translate('Education'));
 ?>
 <section class="section">
     <div class="container">
-        <h1><?= $course->getCode() ?> - <?= $course->getName() ?></h1>
-
-        <ul>
-            <li><strong>Course url:</strong> <a href="<?= $course->getUrl() ?>">
```

Comment:
I would keep this strong. In the new version the URL appears out of nowhere. Context is useful for the user.

Reply:
Mistake on my part, I had originally put the URL in the `<h1></h1>` but in the end that didn't look right. I'll add it back,.